flight/Modules/GimbalControl/gimbalcontrol.c

```diff
@@ -72,9 +76,12 @@ int32_t GimbalControlInitialize(void)
 
 	// Create object queues
 	queue = PIOS_CAN_RegisterMessageQueue(pios_can_id, PIOS_CAN_GIMBAL);
+	queue_roll_pitch = PIOS_CAN_RegisterMessageQueue(pios_can_id, PIOS_CAN_ATTITUDE_ROLL_PITCH);
```

Comment:
Would it be reasonable to rename ```queue``` at the same time? When one has a shortened name and the other a long name it feels like the short name is somehow special. This isn't the intent of the code, AFAICT.